extensions/amp-form/0.1/test/test-amp-form-textarea.js

```diff
@@ -37,9 +38,18 @@ describes.realWin('amp-form textarea[autoexpand]', {
     sandbox = env.sandbox;
   });
 
-  describe('AmpFormTextarea', () => {
+  describe('handleInitialOverflowElements', () => {
     it('should remove autoexpand on elements with initial overflow', () => {
+      const textarea = doc.createElement('textarea');
+      textarea.setAttribute('autoexpand', '');
+      textarea.setAttribute('rows', '1');
+      textarea.setAttribute('cols', '80');
+      textarea.innerHTML = 'big text'.repeat(30);
```

Comment:
While `innerHTML` isn't a problem in tests, better to just use `textContent` when you don't actually need HTML.

Reply:
leaving for a followup PR to align with other test cases